webapp/src/components/Variants/Edition/index.tsx

```diff
@@ -121,6 +134,43 @@ const EditionView = (props: PropTypes) => {
     setCommands(tmpCommand);
   };
 
+  const onCommandImport = (index: number, json: object) => {
+    let tmpCommand: Array<CommandItem> = [];
+    tmpCommand = tmpCommand.concat(commands);
+    // eslint-disable-next-line dot-notation
+    tmpCommand[index].action = (json as any)['action'];
+    // eslint-disable-next-line dot-notation
+    tmpCommand[index].args = { ...((json as any)['args'] as object) };
+    tmpCommand[index].updated = true;
+    setCommands(tmpCommand);
+    enqueueSnackbar(t('variants:importSuccess'), { variant: 'success' });
+  };
+
+  const onCommandExport = (index: number) => {
+    const elm = commands[index];
+    exportJson({ action: elm.action, args: elm.args }, `${elm.id}_command.json`);
+  };
+
+  const onGlobalExport = () => {
+    exportJson(fromCommandItemToJsonCommand(commands), `${studyId}_commands.json`);
```

Comment:
call the back api "GET /studies/{id}/commands", here you export the front content that could contains unsaved commands content. Anyway, the export should call the api, since it's the data truth source.